onboarding/mentors.md

```diff
@@ -3,4 +3,53 @@ title: Onboarding Responsibilities for Mentors of New Hires
 description: How to be an effective mentor of a new hire
 ---
 
-TBD
+# Mentorship at Artsy
+
+### Historical Context
+- TODO: talk to Orta/Sarah/Joey about how we used to do it.
+- [Alan's post](http://artsy.github.io/blog/2016/09/22/professional-development-at-artsy-engineering/)
+
+## Why
+One of Artsy's core values is that [People are Paramount](#). Artsy's engineering team works to reflect this in practice...
+
+## What makes a good mentor (Keep?)
+- A first stop for stupid questions
```

Comment:
🙏 yes keep! And maybe stick stupid in quotes since no questions  are truly stupid 😁